samples/snippets/csharp/VS_Snippets_CLR/conceptual.formatting.overview/cs/iformattable.cs

```diff
@@ -38,22 +38,19 @@ public string ToString(string format)
    
    public string ToString(string format, IFormatProvider provider)  
    {
-      // Handle null or empty arguments.
```

Comment:
I agree that the comments are rather self-evident. At the same time, they're still useful for developers who are new to formatting and IFormattable, and provide a sort of step-by-step set of comments for implementing a ToString method. It would be best to leave them in the example.

Reply:
Considering it being a snippet for documentation purposes I can understand your reasoning.